lib/Canto/TrackDB/Strain.pm

```diff
@@ -58,8 +57,8 @@ __PACKAGE__->add_columns(
   { data_type => "integer", is_foreign_key => 1, is_nullable => 0 },
   "strain_name",
   { data_type => "text", is_nullable => 0 },
-  "strain_abbreviation",
-  { data_type => "text\n       sraing", is_nullable => 1 },
+  "sraing",
```

Comment:
I think this is where the "sraing" value first appears. I'm guessing this isn't intentional?